src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -375,29 +409,7 @@ await diagnosticsWithSameSpan.OrderByDescending(d => d.Severity)
                 supportedScopes, diagnostics.First());
             result.Add(codeFix);
         }
-
-        internal FixAllState CreateFixAllState(
```

Comment:
why was this internal before?

Reply:
I searched both FSharp and TypeScript repos and didnot find usage for the CreateFixAllState